src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,11 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+
```

Comment:
redundant empty line

Reply:
I added new commit named "optimize the code". Am i create everything all right?  Do u see my fixes

Reply:
yep, you can check that on current page 
![image](https://user-images.githubusercontent.com/58385506/88627313-ca99f700-d0b4-11ea-91a1-06cc24a894d7.png)